tests/simulator/test_initialization.py

```diff
@@ -582,5 +583,79 @@ def test_overlapped_particledatas_have_identical_particles(self, interp_order, r
 
 
 
+
+
+    def _test_levelghostparticles_have_correct_split_from_coarser_particle(self, dim, interp_order, refinement_boxes):
+        print("test_levelghostparticles_have_correct_split_from_coarser_particle for dim/interp : {}/{}".format(dim, interp_order))
+
+        datahier = self.getHierarchy(interp_order, refinement_boxes, "particles", cells=30)
+        from pyphare.pharein.global_vars import sim
+        assert sim is not None
+        assert len(sim.cells) == dim
+
+        def domainParticles_for(ilvl):
+            patch0 = datahier.levels()[ilvl].patches[0]
+            particles_ids = [key for key in patch0.patch_datas.keys() if key.endswith("particles")]
+            particlePatchDatas = {k:[] for k in particles_ids}
+            for patch in datahier.levels()[ilvl].patches:
+                for k, v in patch.patch_datas.items():
+                    particlePatchDatas[k].append(v)
+            return { particles_id :
+                aggregate_particles([
+                  patchData.dataset.select(patchData.box) for patchData in patchDatas
+                ]) # including patch ghost particles means duplicates
+                for particles_id, patchDatas in particlePatchDatas.items()
+            }
+
+
+        self.assertTrue(len(level_ghost_boxes(datahier).items()))
+        for ilvl, particle_gaboxes in level_ghost_boxes(datahier).items():
+            self.assertTrue(ilvl > 0) # has no level 0
+            coarse_particles = domainParticles_for(ilvl - 1)
+
+            self.assertTrue(all([particles.size() > 0 for k, particles in coarse_particles.items()]))
+
+            coarse_split_particles = {k: particles.split(sim) for k, particles in coarse_particles.items()}
+
+            for k, particles in coarse_particles.items():
+                self.assertTrue(coarse_split_particles[k].size() > 0)
+                self.assertTrue(coarse_split_particles[k].size() == particles.size() * sim.refined_particle_nbr)
+
+            for gabox in particle_gaboxes:
+                gabox_patchData = gabox["pdata"]
+                particles_id = gabox_patchData.pop_name + "_particles"
```

Comment:
I think this works only because you have all boxes for all populations, right? I mean that if we would stop duplicating the boxes per population that would fail ?

Reply:
ghost boxes imply data IMO, so I don't see it as duplication.

Reply:
there's probably a simpler way about it tho as I guess you're thinking of

Reply:
perhaps we can discuss and make a new issue with a strategy